src/WWT.Providers/Services/OctTileMapBuilder.cs

```diff
@@ -60,11 +60,15 @@ public class OctTileMapBuilder([FromKeyedServices(Constants.ActivitySourceName)]
 
     private async Task<SdssImage?> LoadImage(double raLeft, double decTop, double raRight, double decBottom, bool dr12, CancellationToken token)
     {
-        var innerPath = dr12 ? "DR12" : "dr6";
+        var innerPath = dr12 ? "dr12" : "dr6";
         var raCenter = (raLeft + raRight) / 2.0;
         var decCenter = (decBottom + decTop) / 2.0;
         var scale = Math.Abs(decBottom - decTop) / 500.0;
-        var address = $"https://skyservice.pha.jhu.edu/{innerPath}/imgcutout/getjpeg.aspx?ra={raCenter}&dec={decCenter}&scale={scale * 3600.0}&width={512.0}&height={512.0}&opt=&query=";
+        var updatedScale = scale * 3600.0;
```

Comment:
it seems more readable to just have the `scale` variable and not additional `updatedScale` since that final value is the only one used

Reply:
@twsouthwick Agreed, I got ride of `updatedScale`